itest/assets_test.go

```diff
@@ -943,13 +943,7 @@ func closeAssetChannelAndAssert(t *harnessTest, net *NetworkHarness,
 	t.Logf("Channel closed with txid: %v", closeTxid)
 	t.Logf("Close transaction: %v", spew.Sdump(closeTx))
 
-	// TODO(guggero): Remove this if once the receiver of a channel imports
-	// the proofs correctly as well.
-	if localTapd.node.Name() != "Yara" {
-		waitForSendEvent(
-			t.t, sendEvents, tapfreighter.SendStateComplete,
-		)
-	}
+	waitForSendEvent(t.t, sendEvents, tapfreighter.SendStateComplete)
 
```

Comment:
linter is hitting a weird error
```
go: github.com/lightninglabs/lightning-terminal imports
	github.com/lightningnetwork/lnd/kvdb imports
	github.com/lightningnetwork/lnd/kvdb/etcd imports
	go.etcd.io/etcd/server/v3/embed: mkdir /home/runner/go/pkg/mod/cache/download/go.etcd.io/etcd/server: permission denied
make: *** [Makefile:227: mod-check] Error 1
``` 

don't think it's related to these changes

Reply:
Yeah, it's been failing for a while, but can't reproduce locally. Probably something weird with the GitHub runner itself.